src/utils/transactions.ts

```diff
@@ -82,7 +88,7 @@ export function normalizeGasLimit<T extends GasLimitOptions>(
   options: T,
 ): NormalizeGas<T> {
   const { gas, gasLimit, ...rest } = options;
-  if (gas != null && gasLimit != null) {
+  if (gas && gasLimit) {
```

Comment:
Shorthand for existence checks here might run into confusion with `0` here. Should prefer more explicit form here.